docs/zh/latest/how-to-build.md

```diff
@@ -331,3 +331,15 @@ systemctl stop apisix
 ```
 
 如果通过其他方法安装，可以参考 [配置文件模板](https://github.com/api7/apisix-build-tools/blob/master/usr/lib/systemd/system/apisix.service) 进行修改，并将其放置在 `/usr/lib/systemd/system/apisix.service` 路径下。
+
+## 升级Apache APISIX
+### CentOS 7升级方案
+> 安装方式使用的是`RPM 仓库安装（CentOS 7）`，就可以使用如下方案升级APISIX，在升级之前先关闭`manager-api`
+> 虽然配置文件不会被覆盖，但是最好你还要备份`ETCD、APISX配置文件`
+
+```
+# apisix quit
+# yum search apisix
```

Comment:
`yum search apisix` not really pretty good, it just to find `apisix` or not.

`yum info apisix` maybe better, it can show the updated version.

```shell
[root@lvs ~]# yum info apisix
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
Installed Packages
Name        : apisix
Arch        : x86_64
Version     : 2.12.0
Release     : 0.el7
Size        : 8.5 M
Repo        : installed
From repo   : release
Summary     : Apache APISIX is a distributed gateway for APIs and Microservices, focused on high performance and reliability.
URL         : http://apisix.apache.org/
Licence     : ASL 2.0
Description : Apache APISIX is a distributed gateway for APIs and Microservices, focused on high performance and reliability.

Available Packages
Name        : apisix
Arch        : x86_64
Version     : 2.13.1
Release     : 0.el7
Size        : 2.2 M
Repo        : release/7/x86_64
Summary     : Apache APISIX is a distributed gateway for APIs and Microservices, focused on high performance and reliability.
URL         : http://apisix.apache.org/
Licence     : ASL 2.0
Description : Apache APISIX is a distributed gateway for APIs and Microservices, focused on high performance and reliability.

```